src/components/Section.js

```diff
@@ -10,6 +10,6 @@ export default class Section {
   }
 
   addItem(element) {
-    this._container.prepend(element);
+    this._container.append(element);
```

Comment:
Needs correction: please use `prepend` method here as the new cards should be added to the beginning of the container

Reply:
Still needs correction